pages/account-deleted.tsx

```diff
@@ -0,0 +1,17 @@
+import TopNav from "components/organisms/TopNav/top-nav";
+
+const AccountDeletedPage = () => {
+  return (
+    <div className="min-h-screen flex flex-col">
+      <TopNav />
+      <main className="page-container flex flex-col m-4 pt-28 items-center">
+        <div className="info-container container w-full">
+          <h1 className="text-2xl">Account Deleted</h1>
+          <p>Your account has been deleted.</p>
```

Comment:
Should we do a hard reload and redirect them back to `/` after coming here?

Reply:
Sounds good. On it!

Reply:
Done in 2d7c6bcb9f9db6b263c66c9b114a64cc4b63a4fb